examples/Training/python/Task_Scripts/Raw_Data_Task.py

```diff
@@ -119,6 +89,9 @@
 
 
 message = "Tile average value: %s" % average
-client.setOutput("Message", rstring(message))
-client.setOutput("Kymograph", robject(fileAnn._obj))
-client.closeSession()
+
+
+# Close connection:
+# =================================================================
+# When you are done, close the session to free up server resources.
+conn._closeSession()
```

Comment:
why do we use privet method if there is a proper one, see https://github.com/openmicroscopy/openmicroscopy/pull/4803?